students/myeonghee/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+class User(models.Model):
+
+    name        = models.CharField(max_length=45)
+    email       = models.EmailField()
+    password    = models.CharField(max_length=16)
+    cell_number = models.CharField(max_length=13)
+    address     = models.CharField(max_length=50)
+    birthday	= models.DateField("")
+    sex	        = models.CharField(max_length=2)
```

Comment:
```suggestion
class User(models.Model):
    name        = models.CharField(max_length=45)
    email       = models.EmailField()
    password    = models.CharField(max_length=16)
    cell_number = models.CharField(max_length=13)
    address     = models.CharField(max_length=50)
    birthday	= models.DateField()
    sex	        = models.CharField(max_length=2)
```
좋습니다 명희님! max_length는 넉넉히 설정해주시는 게 좋습니다. DateField의 속성으로 아무것도 주고 싶지 않을 때는 비워두시면 됩니다.